### DISCUSSION TITLE
ylim on plots for time series in decomposition and results

### DISCUSSION THREAD

**Discussion by raoulcollenteur** [pastas/pastas#268] [Q&A] [answered]
Reported by @FransSchaars. The bounds are now taken as the min and max, it would be nice if this is min/max + 10% of the range of the fluctuation.

StepModel shows the problem with the bounds nicely. See following figure. 

![image](https://user-images.githubusercontent.com/12051167/104596283-16952400-5674-11eb-905b-6a0da934aa41.png)

**Comment by rubencalje** [comment]
Yes, it is a problem with the step-model, but for all the other graphs I do not want unnecessary whitespace. Maybe we can make the extra whitespace optional?

**Comment by rubencalje** [comment, marked as answer]
In ml.plots.decomposition there already is the optional argument min_ylim_diff, which makes sure every subplot has a certain height. In your example you could set this to 1 meter:

`ml.plots.decomposition(min_ylim_diff=1.0)`

**Comment by raoulcollenteur** [comment]
Cool thanks that works very well! Gonna change this to a discussion and highlight your answer for future reference!
